tests/modelbench_tests/test_run.py

```diff
@@ -147,15 +158,22 @@ def test_benchmark_anonymous_run_produces_json(self, runner, tmp_path):
 
     def test_nonexistent_benchmarks_can_not_be_called(self, runner):
         result = runner.invoke(cli, ["benchmark", "--benchmark", "NotARealBenchmark"])
-        assert result.exit_code == 2
-        assert "Invalid value for '--benchmark'" in result.output
+        assert result.exit_code == 1
+        assert "No benchmark registered for NotARealBenchmark" in str(result.exception)
```

Comment:
Oh I didn't realize that. It was already throwing that assertion exception before, but I can change it to stop execution more cleanly if you'd like.

Reply:
Ah, ok. If there was an exception before and an exception after, then I think fixing the behavior is optional. But the test should stay in terms if the output, so that when somebody fixes this later, they don't have to change the test back.

Reply:
Fixed